app/src/main/res/xml/search_widget_info.xml

```diff
@@ -20,7 +20,7 @@
 <appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
     android:initialKeyguardLayout="@layout/search_widget"
     android:initialLayout="@layout/search_widget"
-    android:minWidth="250dp"
```

Comment:
Design wanted a 5x1 default widget so I have upped the value here and when I test on smaller devices where 4x1 is the max widget width size it all seems to just render fine. The `minWidth` is more of a default width since `minResizeWidth` is the true minimum but the change feels a bit risky for smaller devices. Can you double check across a few devices to make sure too?